src/rh-footer-links.ts

```diff
@@ -53,16 +53,19 @@ export class RhFooterLinks extends LitElement {
   }
 
   public updateAccessibility() {
-    // set the role for this element to be a list item
-    this.setAttribute('role', 'list');
     // ensure we've rendered to our shadowroot
```

Comment:
I got confused by this comment. I was expecting the next two lines to query the shadow root.

Reply:
yeah, that's old and should be removed. good catch!